task3/task3/Program.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+
+namespace task3
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine(MathFunction("x"));
```

Comment:
И далее нужно распечатать результат.
```suggestion
            Console.WriteLine(MathFunction(x));
```